src/NuGet.Core/NuGet.Protocol.Core.Types/DownloadResourceResult.cs

```diff
@@ -7,14 +7,31 @@
 
 namespace NuGet.Protocol.Core.Types
 {
+    public enum DownloadResourceResultType
```

Comment:
This should be `DownloadResourceResultStatus` postfixing with type in .NET typically alludes to a type from the type system